config/config.go

```diff
@@ -0,0 +1,27 @@
+package config
```

Comment:
I think a new environment variable should be inside each project/ or inside logger config(inside logger folder)